napari_aicssegmentation/widgets/collapsible_box.py

```diff
@@ -0,0 +1,97 @@
+from pathlib import Path
+
+from qtpy.QtGui import QPixmap
+from qtpy.QtWidgets import (
+    QFrame,
+    QHBoxLayout,
+    QLabel,
+    QVBoxLayout,
+    QWidget
+)
+
+DIR = Path.cwd() / "napari_aicssegmentation"
+
+
+"""
+A collapsible box widget containing a title box and a content box. The title box can be clicked
+to toggle the visibility of the content box.
+
+Params:
+    title:      String
+    content:    QLayout to be nested inside the content box
+    isOpen:     Boolean, whether the widget is open or collapsed
+"""
+
+
+class CollapsibleBox(QWidget):
+    def __init__(self, title, content, isOpen=True):
+        super().__init__()
+        self.title = title
```

Comment:
These dont need to be public fields, might be clearer to make them variables and pass them to `_create_title_box()` and `create_content_box()`

Reply:
Hm...... I wrote it this way so that we don't have to pass them as parameters to the functions? @Aerendel do you have an opinion on this?

Reply:
I'm fine with either - but if these are not meant to be public fields you can prefix them with underscore.

Reply:
Cool, I made some of the private, some of them regular variables, and left some public because they need to be accessed in the tests